src/style.css

```
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+
+  --star-size: 16px;
+  --img-star-inactive: url(images/star.svg);
+  --img-star-active: url(images/star-active.svg);
+}
+
+.stars__block {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+}
+
+.stars__star {
+  margin-right: 4px;
+  background-image: var(--img-star-inactive);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: var(--star-size);
+  width: var(--star-size);
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:nth-child(1),
+.stars--2 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n + 3),
+.stars--4 > .stars__star:nth-child(-n + 4),
+.stars--5 > .stars__star:nth-child(-n + 5) {
+  background-image: var(--img-star-active);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: var(--star-size);
+  width: var(--star-size);
+}
```

1. You already have a `width` and `height` styles for star
2. Let's use `background:` shorthand.

```suggestion
.stars--1 > .stars__star:nth-child(1),
.stars--2 > .stars__star:nth-child(-n + 2),
.stars--3 > .stars__star:nth-child(-n + 3),
.stars--4 > .stars__star:nth-child(-n + 4),
.stars--5 > .stars__star:nth-child(-n + 5) {
  background: var(--img-star-active) no-repeat center;
}
```